simcase-game/src/components/Header.js

```diff
@@ -34,41 +34,49 @@ const menuButton = {
 };
 
 const menuList = {
-  textAlign: "center",
+  cursor: "pointer",
   padding: "20px 0",
   margin: 0,
-  cursor: "pointer",
+  textAlign: "center",
   listStyle: "none"
 };
 
 const buttonList = {
-  fontSize: "2em",
   display: "flex",
   flexDirection: "column",
-  alignItems: "center"
+  alignItems: "center",
+  fontSize: "2em"
 };
 
 const buttonOptions = {
   textDecoration: "none",
   lineHeight: "2em",
   color: "#030303",
+  backgroundColor: "blue",
   margin: 15,
-  width: 100,
-  height: 40
+  width: 120,
+  height: 50
 };
 
 const dialogStyles = {
-  margin: "50 auto",
   padding: 0,
-  width: "50vw",
-  height: "70vh",
+  margin: 0,
+  width: "100vw",
+  height: "100vh",
   borderRadius: 10,
-  background: "84b1ff"
+  backgroundColor: "rgba(0, 0, 255, 0.7)"
```

Comment:
This one is opacity.
Blur is this one: https://developer.mozilla.org/en-US/docs/Web/CSS/filter